python/MOCK_AIS.py

```diff
@@ -155,7 +217,17 @@ def speedup_callback(self, msg):
     r = rospy.Rate(1.0 / ais_env.publishPeriodSeconds)  # hz
 
     while not rospy.is_shutdown():
+        #rospy.loginfo(ais_env.real_ships)
+        if ais_env.use_real_ships:
```

Comment:
Agreed it may be best to have a different file for the real boats to keep it clean. Can add an `if` in the launch based on the arguments to use either fake or real